traits/has_traits.py

```diff
@@ -2034,7 +2039,15 @@ def configure_traits(
                 with open(filename, "rb") as fd:
                     self.copy_traits(pickle.Unpickler(fd).load())
 
-        if edit:
+        if edit is not None:
```

Comment:
This still isn't right, I'm afraid: if the user passes in `edit=True`, then `view_application` should be called.  Right  now, `view_application` only gets called if the user passes in `edit=None`, or doesn't pass `edit` at all.

Reply:
Yeah, that was wrong. I've edited this. See if this is good? Thanks for putting up with me :smiley: